posthog/settings/feature_flags.py

```diff
@@ -12,7 +12,9 @@
 default_flag_persistence = [
     # Add hard-coded feature flags for static self-hosted releases here
     "new-paths-ui-edge-weights",
-    "new-sessions-player-events-list",
+    "5730-funnel-horizontal-ui",
+    "938-breakdown-by-multiple-properties",
```

Comment:
CC @pauldambra

Reply:
This had really fallen off my radar, sorry! It's only on for PostHog Team at the moment.

Feels safer to not include it in the release but enable it for beta users or a percentage of cloud?

Reply:
Makes sense, let's do this!